_events/sr2023/second-challenge-submission-deadline.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Second Challenge Submission Deadline
+date: 2023-02-04 20:00:00 Europe/London
+layout: event
+type: challenge-submission
+location: Discord
+---
+
+There are three [challenges][rules-challenges] which teams may optionally
+complete during the competition year in order to earn additional league points.
+The challenges may be approached in any order.
+
+Teams which complete one of the challenges by [8pm GMT on January 7th 2023][first-deadline]
+will earn 8 league points.
+
+Teams which complete a different challenge by 8pm GMT on February 4th 2023 will
```

Comment:
I think that's 8pm BST

Reply:
BST in 2023 starts on March 26th, so I think this is right as-is.

Reply:
D'oh for some reason I thought it said April!